aten/src/ATen/native/native_functions.yaml

```diff
@@ -2238,6 +2238,18 @@
   variants: function, method
   device_guard: False
 
+- func: to_sparse(Tensor self, int64_t sparseDims) -> Tensor
```

Comment:
maybe change `sparseDims` -> `sparse_dim` to be consistent with new API in https://github.com/pytorch/pytorch/pull/13001